### PR TITLE
Use webpack chunks

### PR DESCRIPTION
**Pull request by iHiD**
Spit the bundle into chunks. We can then conditionally load things to speed stuff up. I'll extra things further later.

### CODE REVIEW

**Review by ErikSchierboom — APPROVED** [review]
Nice. This will definitely help improve loading times

**Review by kntsoriano — APPROVED** [review]
🚀

**Review by ErikSchierboom — APPROVED** [review]

**Comment by ErikSchierboom** [comment]
Tests are failing unfortunately.

File: app/helpers/layout_helper.rb [hunk 1]
```diff
@@ -9,4 +9,15 @@ def body_class
...
     classes.join(" ")
   end
+
+  def js_packs
+    packs = ['application']
+    packs << 'internal' if user_signed_in?
+    if (namespace_name == "tracks" && controller_name == "exercises" && action_name == "edit") ||
+       (namespace_name == "test" && controller_name == "editor" && action_name == "show")
+      packs << 'editor'
+    end
```
Comment (ErikSchierboom) [inline app/helpers/layout_helper.rb]: I'd be tempted to move the condition in a separate method, something like:

```suggestion
    packs << 'editor' if editor_displayed?
```

**Comment by iHiD** [comment]
Green 🦜

**Pull request merged**